sanic/deprecated.py

```diff
@@ -0,0 +1,100 @@
+# NOTE(tomaszdrozdz): remove in version: 21.3
+# We replace from_envvar(), from_object(), from_pyfile() config object methods
+# with one simpler update_config() method.
+# We also replace "loading module from file code" in from_pyfile()
+# in a favour of load_module_from_file_location().
+# Please see pull request: 1903
+# and issue: 1895
+import types
+
+from warnings import warn
+
+from sanic.exceptions import PyFileError
+from sanic.helpers import import_string
+
+
+def from_envvar(self, variable_name):
+    """Load a configuration from an environment variable pointing to
+    a configuration file.
+
+    :param variable_name: name of the environment variable
+    :return: bool. ``True`` if able to load config, ``False`` otherwise.
+    """
+
+    warn(
+        "Using `from_envvar` method is deprecated and will be removed in v21.3, use `app.update_config` method instead.",
+        DeprecationWarning,
+        stacklevel=2,
+    )
+
+    config_file = os_environ.get(variable_name)
+    if not config_file:
+        raise RuntimeError(
+            "The environment variable %r is not set and "
+            "thus configuration could not be loaded." % variable_name
+        )
+        return self.from_pyfile(config_file)
+
+
+def from_pyfile(self, filename):
```

Comment:
could we use **annotations** here?

Reply:
No idea what Do You mean :P :)

Reply:
You can check [this link](https://www.python.org/dev/peps/pep-3107/)

Also, in simple words, annotations will allow us to have a better understanding of what datatype we're retrieving, and what datatype function returns, etc.

Reply:
OK.  
But You mean type annotations like:  

    def from_pyfile(self, filename: str):  

or "descriptive anotations" coresponding to what we already have in docstring for this method:  

    def from_pyfile(self, filename: "an absolute path to the config file"):  

??

Reply:
I meant the first option above.

```python
def from_pyfile(self, filename: str):  
```

Reply:
OK.